Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -2704,10 +2702,11 @@ void GStreamerMediaEndpoint::updatePtDemuxSrcPadCaps(GstElement* ptDemux, GstPad
     auto s = gst_caps_get_structure(caps.get(), 0);
     auto s2 = gst_caps_get_structure(currentCaps.get(), 0);
     for (int j = 0; j < gst_structure_n_fields(s2); j++) {
-        const char* name = gst_structure_nth_field_name(s2, j);
-        if (!g_str_equal(name, "media") && !g_str_equal(name, "payload") && !g_str_equal(name, "clock-rate") && !g_str_equal(name, "encoding-name"))
+        auto name = CStringView::unsafeFromUTF8(gst_structure_nth_field_name(s2, j));
+        if (name != "media"_s && name != "payload"_s && name != "clock-rate"_s && name != "encoding-name"_s)
             continue;
-        gst_structure_set_value(s, name, gst_structure_get_value(s2, name));
+        const char* nameString = name.utf8();
+        gst_structure_set_value(s, nameString, gst_structure_get_value(s2, nameString));
```

Comment:
`name.utf8()` should work in the second line as well! Why are you doing this? If it's about performance, you don't need to worry. It should be inlined.